### ISSUE TITLE
all mods RC will try to leave the current map when change map (asteroid vs mars surface)

### ISSUE THREAD

**Issue by Zordhasapie** [ChoGGi/SurvivingMars_CheatMods#54]

**Comment by Zordhasapie** [comment]
thank's for making these mods and sorry because of my terrible english
this is how it look when change map:
![image](https://user-images.githubusercontent.com/67778554/199918002-5561dec0-dc1c-4d43-bc92-ef484354f3df.png)
![image](https://user-images.githubusercontent.com/67778554/199918074-261fb405-a630-4382-b5af-6209654feae3.png)

**Comment by Zordhasapie** [comment]
also, when turn on auto rocket landing and call roket from earth, when it arrival and current play on asteroid map, that roket will land somewhere outside of all map, but still can see on Pin UI

**Comment by ChoGGi** [comment]
Do you have log files from this play session?
https://steamcommunity.com/sharedfiles/filedetails/?id=2391106222

**Comment by Zordhasapie** [comment]
yes i do have it, but i don't think it gonna help, log file look weird some how...
[MarsGOG.exe-20221104-14.33.08-627935d6.log](https://github.com/ChoGGi/SurvivingMars_CheatMo
[MarsGOG.exe-20221104-14.59.44-627935d6.log](https://github.com/ChoGGi/SurvivingMars_CheatMods/files/9942738/MarsGOG.exe-20221104-14.59.44-627935d6.log)
[MarsGOG.exe-20221104-15.00.22-627935d6.log](https://github.com/ChoGGi/SurvivingMars_CheatMods/files/9942739/MarsGOG.exe-20221104-15.00.22-627935d6.log)
ds/files/9942737/MarsGOG.exe-20221104-14.33.08-627935d6.log)

**Comment by ChoGGi** [comment]
One of those is a proper log, looks like the one from when it happens, no errors msgs though I'll have to take a look. The other two logs are from you opening the mod editor.

> Fix Sol 2983
You shouldn't have that mod enabled unless your save is past Sol 2981 (it says that in the description). You also don't need my "math. Functions" mod, I've never seen any mod using that as a dependency.

**Comment by Zordhasapie** [comment]
ok thank you for help, i really appreciate that, thank's a lot

**Comment by ChoGGi** [comment]
I couldn't reproduce your issue (that's way too many mods for me to bother testing), but I did find some other issues with the miner mod. Let me know if it still happens.
